Api/RestCompanyInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2016 Avalara, Inc.
```

Comment:
@LordZardeck - All new files we create should reflect a 2018 copyright year.  (Updating the year in _all_ files is something I should capture as an action item for this release, but for now let's at least update it in newly created files.

Reply:
To that point, I probably need to be using this copy right throughout instead of the standard Classy Llama one, I'll change that